#!/usr/bin/env python
PACKAGE = "dynamic_params"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("re_plan_hil_param",    bool_t,    0, "Toggles ltl re-plannig to avoid re-planning during hil control", True)

exit(gen.generate(PACKAGE, "dynamic_params", "DP"))